lenses/ssh.aug

```diff
@@ -105,11 +105,26 @@ module Ssh =
     let host = [ key /Host/i . spc . value_to_eol . eol . entry* ]
 
 
+   let condition_entry =
+    let value = store  /[^ \t\r\n=]+/ in
+    [ spc . key /[A-Za-z0-9]+/ . spc . value ]
+
+   let match_cond =
+     [ label "Condition" . condition_entry+ . eol ]
+
+   let match_entry = entry
+
+   let match =
+     [ key /Match/i . match_cond
+        . [ label "Settings" .  match_entry+ ]
+     ]
+
+
```

Comment:
Or maybe just `let match = Sshd.match` so we reuse the logic from the other lens. What do you think @lutter?

Reply:
It doesn't seem to work... the definition of eol in ssh.aug and sshd.aug are different... an induce a change in condition_entry ( [^ \t\r\n] vs [^ \t\n])
not sure how to solve that as I'm just discovering augeas... aside from trying to rewrite ssh.aug and sshd.aug.

Reply:
Ok then. Thanks for trying it!